src/mame/drivers/cyclemb.cpp

```diff
@@ -401,6 +424,17 @@ void cyclemb_state::cyclemb_bankswitch_w(uint8_t data)
 	membank("bank1")->set_entry(data & 3);
 }
 
+void cyclemb_state::skydest_bankswitch_w(uint8_t data)
+{
+	membank("bank1")->set_entry(data & 3);
+	flip_screen_set((data & 0x40) == 0);
+}
+
+void cyclemb_state::cyclemb_screen_display_w(uint8_t data)
+{
+	m_screen_display = (data != 0);
```

Comment:
which bit? if multiple, maybe one for sprites, and one for the tilemap?

Reply:
It is only 0 and 1 are written in game, and both sprite and tilemap are hide.

Reply:
Changed to bit0 only.